tests/console/snapper_cleanup_timeline.pm

```diff
@@ -36,9 +36,13 @@ sub convert2numeric {
     return $str2convert;
 }
 
-sub get_used_disk_space {
-    my $used_space = script_output("df -h /  | awk \'NR==2 {print \$5}\'");
-    return convert2numeric($used_space);
+# returns the total disk size number without units, if the given argument is "size".
+# Otherwise it returns the percentage % of used disk space.
+sub get_disk {
```

Comment:
I would propose to still have 2 methods: `get_disk_size` and `get_used_disk`. Not to repeat any code we still can have method like this one which accepts column number as an argument, so get_disk_size will call it with `2` and `get_used_disk` as `5`.

Reply:
Do you mean creating something like 2 functions and one subroutine?

Reply:
Functions and subroutines are synonyms. Point which I don't like is how method is called, to use it you need to know which string to send and it's hardcoded everywhere. So if you would use named boolean arguments here, I'm also fine with that.
Or 3 methods with abstraction, so I won't need to know which parameters to send.

Reply:
Done